etc/config.xml

```diff
@@ -9,6 +9,23 @@
                 <use_logrocket>0</use_logrocket>
                 <logrocket_identify>0</logrocket_identify>
             </general>
+            <issue_grouping>
+                <strip_static_content_version>1</strip_static_content_version>
+                <strip_store_code>1</strip_store_code>
+            </issue_grouping>
```

Comment:
Setting default values here will already override deployment config, it's probably best to leave them empty as well